pkg/runctl/controller.go

```diff
@@ -141,7 +154,9 @@ func (c *Controller) processNextWorkItem() bool {
 		}
 		// Run the syncHandler, passing it the namespace/name string of the
 		// Foo resource to be synced.
-		log.Printf("process %s queue length: %d", key, c.workqueue.Len())
+		if _, finished := finishedRuns.Load(key); !finished {
+			log.Printf("process %s queue length: %d", key, c.workqueue.Len())
```

Comment:
These printouts are only interesting for debugging purpose. I would expect to see a logging library "klog" that allows to log here with a high verbosity. That's how we do it in the build-syncer I think.

Reply:
Not sure if we want to log those 3 entries only on debug level. If we would distinguish between finished and not finished the could would look like this:
```go
		verbosity := klog.Level(2)
		if _, finished := finishedRuns.Load(key); !finished {
			verbosity = klog.Level(4)
		}
		klog.V(verbosity).Infof("process %s queue length: %d", key, c.workqueue.Len())
```

How do you specify the verbosity? `-v=2` does not work out of the box.

Reply:
I guess you need you need to add the klog flag like here:
https://github.wdf.sap.corp/steward-ci/build-syncer/blob/master/cmd/build-syncer.go#L32

Reply:
Concerning the log level (verbosity): For the API-Service we normally run with "INFO" and we defined that for a write-operation _one_ log line should be written to INFO. For me these log lines clearly look like debug.
But if you say they are important, that is fine by me.

Reply:
I fine tuned the verbosity. Looks good now for me, only the state changes of Pipeline Runs are logged on INFO now.